common-test/src/main/java/io/strimzi/test/k8s/BaseKubeClient.java

```diff
@@ -208,6 +208,11 @@ public ProcessResult exec(String pod, String... command) {
         return Exec.exec(cmd);
     }
 
+    @Override
+    public ProcessResult execInKubeWorkspace(String... command) {
```

Comment:
I think @tombentley commented on this below a bit. This actually runs the commands locally, right? So why not call it just `execLocal` or something like that? With the current name I first though you are running this inside some pod.